internal/controller/route/handler.go

```diff
@@ -555,6 +554,11 @@ func (r *Reconciler) assembleTLSSecret(route *routev1.Route) *corev1.Secret {
 // the routes are sorted in ascending order by .metadata.creationTimestamp field
 func (r *Reconciler) getSameHostRoutes(ctx context.Context, namespace, host string) ([]routev1.Route, error) {
 	sameHostRouteList := &routev1.RouteList{}
+
+	if err := r.List(ctx, sameHostRouteList, client.InNamespace(namespace)); err != nil {
```

Comment:
Why are we adding this section?

Reply:
oh, I forgot to remove this section. I wanted to test sth. thanks for mentioning it. it's resolved now